src/content/docs/dropins/checkout/tutorials/multi-step.mdx

```diff
@@ -0,0 +1,1404 @@
+
+   +  // continue to payment
+   +  const deliveryMethod = getCartDeliveryMethod(data);
+   +  if (!deliveryMethod) return;
+   +  await continueToPayment();
+   }
+   ```
+
+   ```diff
+   async function handleCheckoutUpdated(data) {
+     if (isEmptyCart(data)) {
+       await displayEmptyCart();
+       return;
+     }
+
+     removeEmptyCart();
+
+     // continue to shipping
+     const cartShippingAddress = getCartAddress(data, 'shipping');
+     await continueToShipping(cartShippingAddress);
+
+     if (!cartShippingAddress) return;
+
+     continueToDeliveryBtn.setProps((prev) => ({
+       ...prev,
+       disabled: false,
+     }));
+
+   +  const deliveryMethod = getCartDeliveryMethod(data);
+   +  if (!deliveryMethod) return;
+   +
+   +  continueToPaymentBtn.setProps((prev) => ({
+   +    ...prev,
+   +    disabled: false,
+   +  }));
+   }
+   ```
+
+1. Create the `continueToPayment` function to render all the containers that are relevant to the payment step.
+
+   ```js
+   let billToShipping;
+   let billingForm;
+   let paymentMethods;
+   let placeOrder;
+
+   const continueToPayment = async () => {
+     if (!billToShipping) {
+       billToShipping = await CheckoutProvider.render(BillToShippingAddress, {
+         hideOnVirtualCart: true,
+         onChange: (checked) => {
+           $billingForm.style.display = checked ? 'none' : 'block';
+         },
+       })($billToShipping);
+     }
+
+     if (!billingForm) {
+       billingForm = await AccountProvider.render(AddressForm, {
+         addressesFormTitle: 'Billing address',
+         className: 'checkout-billing-form__address-form',
+         formName: BILLING_FORM_NAME,
+         hideActionFormButtons: true,
+         isOpen: true,
+         onChange: debounce((values) => {
+           setAddressOnCart(values, checkoutApi.setBillingAddress);
+         }, DEBOUNCE_TIME),
+         showBillingCheckBox: false,
+         showShippingCheckBox: false,
+       })($billingForm);
+     }
+
+     if (!paymentMethods) {
+       paymentMethods = await CheckoutProvider.render(PaymentMethods)($paymentMethods);
+     }
+
+     if (!placeOrder) {
+       placeOrder = await CheckoutProvider.render(PlaceOrder)($placeOrder);
+     }
+
+     continueToPaymentBtn.remove();
+     $continueToPaymentBtn.remove();
+   };
+   ```
+
+   <Diagram caption="Continue to payment">
+   ![Continue to payment](@images/dropins/checkout/continue-to-payment.png)
+   </Diagram>
+</Steps>
+
+At this point, you should have a fully functional multi-step checkout, but there's still some work to do. Let's move on to the last step.
+</Task>
+
+<Task>
+### Create order confirmation
+
+The last step to complete the multi-step checkout process is to create an order confirmation page. To make things easy, you can reuse the code from the `commerce-checkout.js` block in the boilerplate template.
+
+<Steps>
+1. Copy the `displayOrderConfirmation` function.
+
+1. Copy the `handleCheckoutOrder` handler.
```

Comment:
Is the ordered list correct?

Reply:
Actually, our static site generator takes care of the numbering. We always specify "1" for maintainability. This is how it looks after the site is built:

![Screenshot 2024-12-11 at 9 55 38 AM](https://github.com/user-attachments/assets/c8c466b3-be77-450f-b539-4072e1fe6214)